script.js

```diff
@@ -5,22 +5,62 @@ var game = {
   players: [],
   hands: [],
   buildDeck: function(){
-
+    for (suit in suits) {
+      for (value in values) {
+        deck.push(values[value].toString() + "-" + suits[suit])
```

Comment:
`value` and `suit` aren't actually values and suits here -- they're indexes. Might name your variables something different.